client/web/compose/src/components/PageBlocks/TabBase.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <wrap
+    :id="`tab-${options.blockIndex}`"
+    v-bind="$props"
+    v-on="$listeners"
+  >
+    <b-tabs
+      v-if="options.tabs.length"
+      v-model="activeTab"
+      v-bind="{
+        align: options.tabStyle.alignment,
+        fill: options.tabStyle.fillJustify === 'fill',
```

Comment:
instead of tabStyle, just call it style since its already belonging to the tab class